graphprot/NeuralNet.py

```diff
@@ -95,29 +95,79 @@ def __init__(self, database, Net,
 
         elif self.task == 'class':
             self.loss = nn.CrossEntropyLoss(weight = self.class_weights, reduction='mean')       
+        
+        
+    def plot_loss(_train_loss, _valid_loss=[], nepoch):        
+    
+        import matplotlib.pyplot as plt
+            
+        if len(_valid_loss) > 1:
+            plt.plot(range(1,nepoch+1), _valid_loss, c='red', label='valid')
+
+        if len(_train_loss) > 1:
+            plt.plot(range(1,nepoch+1), _train_loss, c='blue', label='train')
+            plt.title("Loss/ epoch")
+            plt.xlabel("Number of epoch")
+            plt.ylabel("Total loss")
+            plt.legend()
+            plt.savefig('loss_epoch.png')
+            plt.close()
+            
+            
+    def plot_acc(_train_acc, _valid_acc=[], nepoch):  
```

Comment:
I'm surprised it accept a non keyword argument  (nepoch) after a keyword arg (_valid_acc). Here as well I would remove the '_' from the variable name :)